casepro/msgs/models.py

```diff
@@ -5,6 +5,7 @@
 from dash.orgs.models import Org
 from dash.utils import get_obj_cacheable
 from django.contrib.auth.models import User
+from django.contrib.postgres.fields import HStoreField
```

Comment:
Initially used `JSONField` but that relies on postgresql 9.4 as Django (rightly) uses `jsonb` for the underlying schema definition, `jsonb` was only added in 9.4. Using `HStore` as travis is configured for postgresql 9.3 and so I'm assuming some production systems have that as a requirement.